crates/node-bindings/src/resolver.rs

```diff
@@ -233,38 +192,38 @@ pub struct JsInvalidations {
 #[napi]
 pub struct Resolver {
   mode: u8,
-  #[cfg(not(target_arch = "wasm32"))]
-  resolver: parcel_resolver::Resolver<'static, EitherFs<JsFileSystem, OsFileSystem>>,
-  #[cfg(target_arch = "wasm32")]
-  resolver: parcel_resolver::Resolver<'static, JsFileSystem>,
+  resolver: parcel_resolver::Resolver<'static>,
   #[cfg(not(target_arch = "wasm32"))]
   invalidations_cache: parcel_dev_dep_resolver::Cache,
```

Comment:
Does this need to be conditionally compiled in? Can we supply an empty `Cache` for `wasm` builds?

Reply:
My change actually removes the need for some of the conditions. We can revisit the other stuff later as it's not related to my changes.